OZprivate/rawJS/OZTreeModule/src/api/api_manager.js

```diff
@@ -64,7 +64,7 @@ class APIManager {
     return new Promise((resolve, reject) => {
       api_wrapper({
         method: 'get',
-        url: '/tour/list.json',  // TODO: Config-ise it
+        url: config.api.tour_list_api,  // TODO: Config-ise it
```

Comment:
This TODO is redundant now, you've done it. Otherwise all looks good to me!

Reply:
Good point, thanks! I'll remove and merge